client/app/scripts/controllers/repo.js

```diff
@@ -21,6 +21,16 @@ angular.module('clientApp')
       'done': 'Done'
     };
 
+    // List of build statuses.
```

Comment:
Comment looks wrong. I think it should be something like "List of build possible intervals"